README.md

```diff
@@ -1,279 +1,146 @@
+- [Writing Your First App](support/guide/getting_started/writing_your_first_app.md)
+- [Understanding JAC Programs](support/guide/getting_started/understanding_jac_programs.md)
+
+
+## The JAC Programming Language Guide
+
+- [Language Overview](support/guide/jac_language_guide/jac_language_overview.md)
+- [Grammar](support/guide/jac_language_guide/jac_grammar.md)
+- [Operators](support/guide/jac_language_guide/operators.md)
+- [Control Flow](support/guide/jac_language_guide/control_flow.md)
+- [Collections](support/guide/jac_language_guide/collections.md)
+- [Nodes](support/guide/jac_language_guide/nodes.md)
+- [Edges](support/guide/jac_language_guide/edges.md)
+- [Graphs](support/guide/jac_language_guide/graphs.md)
+- [Walkers](support/guide/jac_language_guide/walkers.md)
+    - [Traversing a Graph](support/guide/jac_language_guide/traversing_a_graph.md)
+    - [Referencing a Node in Context](support/guide/jac_language_guide/referencing_node_context.md)
+- [Passing Arguments](support/guide/jac_language_guide/passing_arguments.md)
+- [Specifying Operating Context](support/guide/jac_language_guide/specifying_operating_context.md)
+- [Actions](support/guide/jac_language_guide/actions.md)
+    - [Date Actions](support/guide/jac_language_guide/date_actions.md)
+    - [File Actions](support/guide/jac_language_guide/file_actions.md)
+    - [Net Actions](support/guide/jac_language_guide/net_actions.md)
+    - [Rand Actions](support/guide/jac_language_guide/rand_actions.md)
+    - [Request Actions](support/guide/jac_language_guide/request_actions.md)
+    - [Standard Actions](support/guide/jac_language_guide/standard_actions.md)
+    - [Vector Actions](support/guide/jac_language_guide/vector_actions.md)
+    - [Jaseci Actions](support/guide/jac_language_guide/jaseci_actions.md) 
+
+
+## Codelabs and Guides
+
+- [Build a Custom Jaseci Module](support/codelabs/custom_jaseci_module_codelab.md)
+- [Building a Conversational AI System](support/codelabs/canonicai/chapter1.md)
+    - [Chapter 2](support/codelabs/canonicai/chapter2.md)
+    - [Chapter 3](support/codelabs/canonicai/chapter3.md)
+    - [Chapter 4](support/codelabs/canonicai/chapter4.md)
+    - [Chapter 5](support/codelabs/canonicai/chapter5.md)
+    - [Chapter 6](support/codelabs/canonicai/chapter6.md)
+    - [Chapter 7](support/codelabs/canonicai/chapter7.md)
+- [Install Jaseci using Helm](support/helmcharts/jaseci/README.md)
+- [Create AWS EKS Infrastructure for Jaseci using Terraform](support/infrastructure/terraform/aws/README.md)
+- [Create Azure AKS Infrastructure for Jaseci using Terraform](support/infrastructure/terraform/azure/README.md)
+- [Locust Load Test for JASECI](support/locust/README.md)
+- [Setting up Monitoring for JASECI](support/monitoring/README.md)
+- [CanoniCAI Example](examples/CanoniCAI/CCAI_codelab.md)
+- [NER Examples](examples/ner_examples/README.md)
+
+
+## Jaseci Library Reference
+
+- [Jaseci Kit](support/guide/jaseci_ai_kit/README.md)
```

Comment:
Can the links here point to the existing md files in `jaseci_ai_kit`? instead of creating duplicate ones under `support`?